samples/net/http_server/boards/nrf7002dk_nrf5340_cpuapp_ns.conf

```diff
@@ -0,0 +1,51 @@
+#
```

Comment:
@simensrostad  do you see a benefit of including also an overlay for the default (Secure) 5340 Application Core build?

Reply:
Not really. We should inspire customers to build for NS. For security reasons.